spec/lib/rspec/sorbet_spec.rb

```diff
@@ -131,6 +131,31 @@ class Animal; end
         it_should_behave_like 'it allows an instance double'
       end
 
+      describe 'with an existing error handler' do
+        let(:handler) { proc {|_,_| raise ArgumentError, 'foo'} }
+
+        before do
+          T::Configuration.call_validation_error_handler  = handler
+          described_class.allow_doubles!
+        end
+
+
+        class PassthroughSig
+          extend T::Sig
+
+          sig { params(message: String).void }
+          def initialize(message)
+            @message = message
+          end
+        end
+
+        specify do
+          # Error is not rasied becasue handler is no-op
```

Comment:
`raised`

Reply:
Updated the comment to correctly reflect the behavior and make the test a bit more solid in it's assertions.